pkg/ingester/ingester_test.go

```diff
@@ -6062,11 +6035,14 @@ func TestIngester_QueryStream(t *testing.T) {
 						return receivedSeries, err
 					}
 
-					actualTimeseries = append(actualTimeseries, resp.Timeseries...)
-					actualChunkseries = append(actualChunkseries, resp.Chunkseries...)
+					if resp.StreamingSeries != nil {
```

Comment:
Why do we need to check for `nil` here and below? `append` and `len` can both handle `nil` slices.

Reply:
Fixed in fc532fda0556bc86a40658c95d18d120e73a67c9